app/code/Magento/Cms/Controller/Adminhtml/Page/Delete.php

```diff
@@ -30,8 +49,7 @@ public function execute()
             $title = "";
             try {
                 // init model and delete
-                $model = $this->_objectManager->create(\Magento\Cms\Model\Page::class);
-                $model->load($id);
+                $model = $this->cmsPage->load($id);
```

Comment:
1. I think in this case it's better to use repository to retrieve and then delete this cms page
2. Loading model is not correct way. It's better to use \Magento\Cms\Model\PageFactory and create one instance of model your your needs (if really needed)

Reply:
Done